compiler/natives/net/http/http.go

```diff
@@ -38,11 +39,22 @@ func (t *XHRTransport) RoundTrip(req *Request) (*Response, error) {
 	xhr.Set("onload", func() {
 		header, _ := textproto.NewReader(bufio.NewReader(bytes.NewReader([]byte(xhr.Call("getAllResponseHeaders").String() + "\n")))).ReadMIMEHeader()
 		body := js.Global.Get("Uint8Array").New(xhr.Get("response")).Interface().([]byte)
+
+		contentLength := int64(-1)
+		if req.Method == "HEAD" {
+			i, err := strconv.ParseInt(header.Get("content-length"), 10, 64)
```

Comment:
Is it correct that `"content-length"` is being used instead of `"Content-Length"`? It might very well be, I just wanted to confirm.